src/docs/01_introduction_and_goals.adoc

```diff
@@ -62,10 +62,10 @@ A table with quality goals and concrete scenarios, ordered by priorities
 [options="header", cols="0,0,4,3"]
 |===
 |Priority| Goal         | Motivation | Scenarios
-|1| Usability |The app must be easy to use by non-technical people. | A new user must be able to understand the application and use it without advanced knowledge in the software engineering field.
-|2| Privacy    | Users must be the owners of their own data. | Users must be able to locate their data wherever they want to and have full control of it.
-|3| Effectiveness | Users must be able to share their routes according to the functionalities of the system. | The routes of a user must be able to be shared through the application.
-|4| Interoperability | The system must be able to communicate with other similar SOLID apps | Routes created with other SOLID apps must be able to be loaded to the system and be able to be worked with.
+|1| Usability |The app must be easy to use by non-technical people. | As a new user, I want to be able to use the app without needing any external help.
+|2| Privacy    | Users must be the owners of their own data. | As a user, I want to be able to choose where I want my own data to be.
+|3| Effectiveness | Users must be able to share their routes according to the functionalities of the system. | As a user, I want to be able to share my routes with my contacts.
+|4| Interoperability | The system must be able to communicate with other similar SOLID apps | As a user, I want to be able to share the routes loaded in my application with other similar SOLID apps.
```

Comment:
@lamasumas how it looks now? Reformulation done.

Reply:
LGTM